commands-yml/commands/element/attributes/replace-value.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Get Element Text
```

Comment:
It seems different commands are mixed here: `/session/:sessionId/element/:elementId/value`, `/session/:sessionId/element/:elementId/text`, `/session/:sessionId/element/:elementId/submit`